### PR TITLE
TDP Activity Search: Mobile customization

### PR DESCRIPTION
**Pull request by karenkitchens**
TDP activity search need to be customized since the desktop version of search had too many options. This will help folks move through the search more effectively when on mobile

---

## Additions
Added mobile classes to html
Added a rule for changing mobile facets

## How to test this PR

1. Go to the TDP Activity search, and scale down to mobile, then refresh the page, and notice if the facets work correctly.

## Notes and todos

Need to move the JS rule to somewhere more appropriate
-

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Future todos are captured in comments and/or tickets

#### Browser testing

Visually tested in the following supported browsers:
- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] Edge 18 (the last Edge prior to it switching to Chromium)
- [ ] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
- [x] Safari on iOS
- [ ] Chrome on Android

#### Accessibility

- [x] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [x] Screen reader friendly
- [x] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other

- [x] Is useable without CSS
- [x] Is useable without JS
- [x] Does not introduce new lint warnings
- [x] Flexible from small to large screens

### CODE REVIEW

**Comment by Scotchester** [comment]
@karenkitchens FYI, I merged the main branch into this PR so that our dev preview deployment would hopefully complete successfully.

**Comment by mrclay** [comment]
@Scotchester FYI we've reworked this a bit and added a test.

File: cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+function setInnerWidth(width) {
+  innerWidth = width;
+}
+
+/**
+ * Alter DOM as necessary before cfpb-expandables ExpandableTransition.init()
+ * so some elements won't be expanded by default.
+ */
+function beforeExpandableTransitionInit() {
+  document.querySelectorAll( '.' + MOBILE_COLLAPSED_CLASS ).forEach( el => {
```
Comment (Scotchester) [inline cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js:22]: I think this line is breaking the functionality of the filters in IE 11. None of the faceting works as it used to, and I see this error in the console:

```
Object doesn't support property or method 'forEach'
```

Comment (Scotchester) [reply cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js:22]: (Affecting desktop widths, just to be clear.)

Comment (mrclay) [reply cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js:22]: @Scotchester I'll look into this right away. Is IE11 the baseline or (gulp) earlier versions.

Comment (Scotchester) [reply cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js:22]: Thanks, Steve! For JS-required functionality like this, we only need to support IE 11.

File: cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js [hunk 2]
```diff
@@ -19,14 +19,17 @@ function setInnerWidth(width) {
  * so some elements won't be expanded by default.
  */
 function beforeExpandableTransitionInit() {
-  document.querySelectorAll( '.' + MOBILE_COLLAPSED_CLASS ).forEach( el => {
+  const nodeList = document.querySelectorAll( '.' + MOBILE_COLLAPSED_CLASS );
+
+  // IE11 lacks NodeList.forEach
+  [].forEach.call(nodeList, el => {
```
Comment (mrclay) [inline cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js:25]: @Scotchester I don't have IE11 to test, but this appears the go-to workaround for this.

Comment (mrclay) [reply cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js:25]: Sorry, I'll fix linting.

**Review by Scotchester — APPROVED** [review]
That did the trick! I'll get this merged and try to deploy it tomorrow.

**Pull request merged**